app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/ValidatorDetailsItem.kt

```diff
@@ -23,11 +24,19 @@ fun ValidatorDetailsItem(validator: ValidatorDetail, modifier: Modifier = Modifi
             modifier = Modifier.size(iconSize),
             validator = validator
         )
-        Text(
-            validator.name,
-            style = RadixTheme.typography.body1Header,
-            color = RadixTheme.colors.gray1,
-            maxLines = 1
-        )
+        Column {
+            Text(
```

Comment:
@raf-rdx I would also add a check here if the validator name is empty to not show the validator's text +
to make the collumn center align its elements just in case